app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,29 @@
+<nav class="bg-white">
+  <div id ='nav' class="flex items-center justify-between content-end flex-wrap bg-white p-5 shadow-lg">
+<!-- Menu hamburguesa-->
+    <div class="block">
+        <button id='boton' class="flex items-center px-3 py-2">
```

Comment:
I recommend putting the name of the menu id, to id = 'menu', to make it compatible with the sidebar.

Reply:
The modal that come out of the menu is here: <div id="menu" class="px-5 pt-2 pb-4 hidden">and its id is "menu" so the id of the menu can't be "menu"

Reply:
In fact, this is how we configured it to adapt it in the JS and make it work.

Reply:
I tried to change the id of the button to "menu" but when I ran the server the button didn't work